docs/api-guide/generic-views.md

```diff
@@ -125,22 +117,22 @@ For example:
 Note that if your API doesn't include any object level permissions, you may optionally exclude the `self.check_object_permissions`, and simply return the object from the `get_object_or_404` lookup.
 
 #### `filter_queryset(self, queryset)`       
-       
+
 Given a queryset, filter it with whichever filter backends are in use, returning a new queryset.   
-        
+
 For example:       
-       
+
     def filter_queryset(self, queryset):
         filter_backends = (CategoryFilter,)
-        
+
         if 'geo_route' in self.request.query_params:
             filter_backends = (GeoRouteFilter, CategoryFilter)
         elif 'geo_point' in self.request.query_params:
             filter_backends = (GeoPointFilter, CategoryFilter)
-        
+
         for backend in list(filter_backends):
             queryset = backend().filter_queryset(self.request, queryset, view=self)
-        
+
```

Comment:
@xordoquy hmmm... can you please explain why it matters? My IDE would have stripped the redundant whitespace automatically which seems pretty sensible to me.